### PR TITLE
Seekers of the Storm compatibility

### PR DESCRIPTION
**Pull request by Phreelosu**
so while the code works, including the artifact, the two items and their functionalities, and the drones themselves

you'll unfortunately still need to regenerate the soundbanks in the 2023 version of WWise, as Gearbox updated WWise too

The language tokens don't really work at the moment, but it most certainly is on TILER2's side, so whenever ThinkInvis feels like updating TILER2, the language tokens will automatically work again as it's technically using R2API's methods

I haven't added support for Sotv or Sots survivors yet, but once the modding scene has recovered, I'll try my best maintaining this mod if I'm allowed to

EDIT: also uses the unpublished version of ChensHelpers, so without that, this won't work

### CODE REVIEW

**Comment by tieeeeen1994** [comment]
I don't think TILER2 will be updated anymore. You might want to remove the dependency.

For the soundbanks, are you able to fix them?

**Comment by tieeeeen1994** [comment]
For making the mod work, please bear with us. 

I still want to maintain the quality I aim for the mod; I'd like to fix the mod into working order where assets are also properly showing.

**Comment by tieeeeen1994** [comment]
About you maintaining the mod; we'd love to! 

The current setup of the fork is good enough. I'd love for the main repository to be kept updated.

**Comment by Phreelosu** [comment]
> I don't think TILER2 will be updated anymore. You might want to remove the dependency.
> 
> For the soundbanks, are you able to fix them?
> 
> Edit: I realize that we may need to fetch the WWISE project... Would it be an easy import?

yes, its basically just opening the WWise project in the 2023 version and regenerating the soundbank, thats all there is to it thankfully

ThinkInvis is active, and is planning to update TILER2 and TinkersSatchel to SotS

**Comment by tieeeeen1994** [comment]
> Yes, its basically just opening the WWise project in the 2023 version and regenerating the soundbank, thats all there is to it thankfully
> 
> ThinkInvis is active, and is planning to update TILER2 and TinkersSatchel to SotS

That is indeed good to hear. I would prefer if you keep the outdated mod support for other mods? Comment it only if you can.

**Comment by cheeeeeeeeeen** [comment]
[chensgradiusmod_soundbank.zip](https://github.com/user-attachments/files/16815913/chensgradiusmod_soundbank.zip)

Hello! Here's the regenerated sound bank file using `Wwise 2023.1.6`.

**Comment by Phreelosu** [comment]
done, ill be onto chilldrone next as DamageTypeAPI is updated o7

File: ChensGradiusMod/Extensions.cs [hunk 1]
```diff
@@ -130,7 +130,7 @@ public static void InitializeDroneModelComponents(this GameObject droneModel, Ch
             characterModel.BuildRendererInfos(droneModel);
             characterModel.autoPopulateLightInfos = true;
             characterModel.invisibilityCount = 0;
-            characterModel.temporaryOverlays = new List<TemporaryOverlay>();
+            characterModel.temporaryOverlays = new List<TemporaryOverlayInstance>(); //I used this bootleg aah method in Titans of the Rift so hopefully it works here too..?
```
Comment (tieeeeen1994) [inline ChensGradiusMod/Extensions.cs:133]: We'd prefer if this is tested.

Comment (Phreelosu) [reply ChensGradiusMod/Extensions.cs:133]: image

according to the "Upgrading mods to SOTS" thread under the #wiki channel on official modding discord, this is the intended way to do it as Direseeker, Arsonist, Lee Hyperreal mods already utilizes this.

Comment (tieeeeen1994) [reply ChensGradiusMod/Extensions.cs:133]: We still want visual confirmation about it. Are you able to setup the locally and see of Chill Drone would appear as normal? If it does, then it works.

Comment (Phreelosu) [reply ChensGradiusMod/Extensions.cs:133]: image
image

no errors in the console either, so I think its a success

**Comment by tieeeeen1994** [comment]
Nice! Approved.

**Review by tieeeeen1994 — APPROVED** [review]

**Comment by cheeeeeeeeeen** [comment]
@Phreelosu Are we ready to upload the mods, or are there more backlogs?

**Comment by Phreelosu** [comment]
> @Phreelosu Are we ready to upload the mods, or are there more backlogs?

we still need to wait for a TILER2 update unfortunately, but it shouldn't take long to be honest

**Pull request merged**